TheMET/TheMET/Network/MetAPI.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  MetAPI.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 25/04/2023.
+//
+
+import Foundation
+
+class MetAPI{
+    
+    private var networkManager: NetworkManager
+    
+    init(networkManager: NetworkManager) {
+        self.networkManager = networkManager
+    }
+    
+    func objects(metadataDate: Date? = nil, departmentIds: [Int] = [], completion: @escaping (ObjectsResponse?) -> Void) {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd"
+        let dateString: String?
+        if let date = metadataDate {
+            dateString = dateFormatter.string(from: date)
+        } else {
+            dateString = nil
+        }
+        let idArrayString: String?
+        if !departmentIds.isEmpty {
+            idArrayString = departmentIds.map {String($0)}.joined(separator: "|")
+        } else {
+            idArrayString = nil
+        }
+        let urlString: String = "https://collectionapi.metmuseum.org/public/collection/v1/objects"
```

Comment:
I suggest to split this string in 2:

- a local `let` property to keep `https://collectionapi.metmuseum.org/`
- a suffix `public/collection/v1/objects` to be added in this method

It will allow you to use the same constant for all the other methods which will be implemented later.

Reply:
fixed